### DISCUSSION TITLE
What are the pdf's supported standards ?

### DISCUSSION THREAD

**Discussion by sebr72** [LibrePDF/OpenPDF#939] [Q&A]
Hi,
Could someone please confirm my understanding:
- Standard PDF/A-1a is deployed in 1.3.30
- Standard PDF/A-1b is deployed in master
- Standard PDF/A-2 is not implemented (is there any work in progress to support it ?)
- Standard PDF/A-3 is not implemented (is there any work in progress to support it ?)

**Comment by asturio** [comment]
These are all moving targets. Creating some tests and documenting these are welcome. We would appreciate someone contributing more PDF/A support in OpenPDF.
